examples/cpp/common/opengl/GlUtils.cpp

```diff
@@ -562,6 +562,7 @@ namespace oria {
     mv.withPush([&]{
       mv.rotate(-HALF_PI - 0.22f, Vectors::X_AXIS).scale(0.5f);
       renderGeometry(shape, program, [&] {
+        Uniform<float>(*program, "ForceAlpha").Set(alpha);
```

Comment:
Fixed.

On Sun, Mar 1, 2015 at 6:55 PM, Brad Davis <notifications@github.com> wrote:

> In examples/cpp/common/opengl/GlUtils.cpp
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/73#discussion_r25569956>
> :
>
> > @@ -562,6 +562,7 @@ namespace oria {
> >      mv.withPush([&]{
> >        mv.rotate(-HALF_PI - 0.22f, Vectors::X_AXIS).scale(0.5f);
> >        renderGeometry(shape, program, [&] {
> > +        Uniform<float>(*program, "ForceAlpha").Set(alpha);
>
> You need to change this so that ForceAlpha is only set if the incoming
> value is not 0. Actually it would be better if the default parameter
> defined in the header was -1 and then only set alpha if it's >= 0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/73/files#r25569956>
> .
>